tests/test_headings.py

```diff
@@ -41,3 +41,31 @@ def test_annotate_headings_works(m_16_19_doc):
 
 def test_main_works(m_16_19_doc):
     headings.main(m_16_19_doc)
+
+
+def test_annotate_headings_m_15_16(m_15_16_doc):
+    """
+    Minor difference in font size were causing many false positives for
+    headings in this document; this test checks that only the actual headings
+    are present.
+    We're looking only at what's after the SUBJECT: line because that line is
+    part of the metadata and should be treated differently.
+    """
+    split_heading = "SUBJECT: Multi-Agency Science and Technology Priorities "\
+        "for the FY 2017 Budget"
+    headers = [str(line).strip() for line in
+               headings.annotate_headings(m_15_16_doc)]
+    if split_heading in headers:
+        subj_index = headers.index(split_heading)
+        relevant = headers[subj_index + 1:]
+    else:
+        relevant = headers
```

Comment:
Hmm, since this is a test that's not doing anything with randomness, why would `split_heading` be in `headers` for some test runs but not for others? Is it possible to get rid of one of the branches of this `if` statement?

Reply:
Premature optimization—I figured that the SUBJECT: and other lines in the metadata might get treated differently in the future and thus not be present… but the test would have to be changed anyway to include the metadata-handling part, so this is probably superfluous.